### PR TITLE
Updates from featuretools v0.26.0

### PR DESCRIPTION
**Pull request by rwedge**
Merges in changes from the 0.26.0 release of featuretools

### CODE REVIEW

**Review by tamargrey — APPROVED** [review]
Woodwork stuff looks good to me, but probably worth waiting on another review to confirm everything looks good!

**Review by thehomebrewnerd — APPROVED** [review]
One tiny release notes thing, but otherwise looks good.

**Review by jeff-hernandez — APPROVED** [review]

**Pull request merged**
